unit-tests/algo/convertions/test-y411.cpp

```diff
@@ -0,0 +1,142 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+//#cmake: static!
+//#cmake:add-file ../../../src/proc/color-formats-converter.cpp
+
+#include "../algo-common.h"
+#include <librealsense2/rsutil.h>
+#include "../src/types.h"
+#include "../src/proc/color-formats-converter.h"
+
+const int W = 16;
+const int H = 2;
+const int BPP_RGB = 24;
+const int BPP_YUV = 24;
+const int BPP_Y11 = 12;
+const int SOURCE_BYTES_Y411 = W * H*BPP_Y11 / 8;
+const int SOURCE_BYTES_YUV = W * H*BPP_YUV / 8;
+const int DEST_BYTES = W * H* BPP_RGB / 8;
+
+struct byte3
+{
+    byte data[3];
+    bool operator==(const byte3& other) const
+    {
+        return (int)data[0] == (int)other.data[0] && (int)data[1] == (int)other.data[1] && (int)data[2] == (int)other.data[2];
+    }
+};
+
+std::ostream & operator<<(std::ostream & os, const byte3& obj)
+{
+    os << (int)obj.data[0] << " " << (int)obj.data[1] << " " << (int)obj.data[2];
+    return os;
+}
+
+void convert_single_yuv_to_rgb(const byte3 yuv, byte3 *rgb)
+{
+    int32_t c = yuv.data[0] - 16;
+    int32_t d = yuv.data[1] - 128;
+    int32_t e = yuv.data[2] - 128;
+
+    int32_t t;
+#define clamp( x ) ( ( t = ( x ) ) > 255 ? 255 : t < 0 ? 0 : t )
+    rgb->data[0] = clamp((298 * c + 409 * e + 128) >> 8);
+    rgb->data[1] = clamp((298 * c - 100 * d - 208 * e + 128) >> 8);
+    rgb->data[2] = clamp((298 * c + 516 * d + 128) >> 8);
+#undef clamp
+}
+
+void convert_yuv_to_rgb(byte3 *rgb, const byte3 * yuv)
```

Comment:
Shouldn't `yuv` be the left argument and `rgb` the right (based on their order in the function name)?